crates/ide_db/src/call_info.rs

```diff
@@ -166,7 +166,7 @@ impl ActiveParameter {
 
         let idx = active_parameter?;
         let mut params = signature.params(sema.db);
-        if !(idx < params.len()) {
+        if params.len() <= idx {
```

Comment:
Don't have strong opinion here, but the original spelling is deliberate. `index < len` is the invariant you want to be held during indexing, and I like to spell the invariat-checking guards as `!invariant`. That is, I think there's a marginal readability benefit in seeing invariant stated in a positive way.

Reply:
I'm usually not fond of these kinds of negations but you are raising a good point there. I'll revert that then I think.